src/test/kotlin/com/atlassian/performance/tools/jiraactions/api/parser/MergingActionMetricsParserTest.kt

```diff
@@ -0,0 +1,49 @@
+package com.atlassian.performance.tools.jiraactions.api.parser
+
+import com.atlassian.performance.tools.io.api.directories
+import com.atlassian.performance.tools.io.api.ensureDirectory
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Test
+import java.io.File
+import java.nio.file.Files
+import java.util.zip.ZipFile
+
+class MergingActionMetricsParserTest {
+
+    @Test
```

Comment:
How to run CI for a particular commit?

Reply:
Unfortunately, neither CircleCI nor TravisCI have that feature. There are workarounds, but only Bitbucket Pipelines had it out of the box. I'll try to apply one of the workarounds tomorrow.